src/Lykke.Service.BlockchainWallets/Controllers/WalletsController.cs

```diff
@@ -38,18 +38,13 @@ public class WalletsController : Controller
         [HttpPost(RouteSuffix + "/by-client-ids/{clientId}")]
         public async Task<IActionResult> CreateWallet([FromRoute] string blockchainType, [FromRoute] string assetId, [FromRoute] Guid clientId, [FromQuery] WalletType? walletType)
         {
-            if (!ValidateRequest(blockchainType, ref assetId, clientId, out var badRequest))
-            {
-                return badRequest;
-            }
+            blockchainType = blockchainType.TrimAllSpacesAroundNullSafe();
+            assetId = assetId.TrimAllSpacesAroundNullSafe();
 
-            if (!await _blockchainIntegrationService.AssetIsSupportedAsync(blockchainType, assetId))
-            {
-                return BadRequest
-                (
-                    BlockchainWalletsErrorResponse.Create($"Asset [{assetId}] or/and blockchain type [{blockchainType}] is not supported.")
-                );
-            }
+            if (!ValidateRequest(out var badRequest,
+                ParamsToValidate.UnsupportedAssetId | ParamsToValidate.EmptyClientId,
```

Comment:
`ParamsToValidate.UnsupportedAssetId | ParamsToValidate.EmptyClientId | ParamsToValidate.EmptyBlockchainType`
Because we used to validate `blockchainType` before all changes.